project.owasp.yaml

```diff
@@ -0,0 +1,39 @@
+audience:
+  - builder
+  - defender
+leaders:
+  - name: Kevin Wall
+    email: kevin.w.wall@owasp.org
+    github: kwwall
+  - name: Matt Seil
+    email: matt.seil@owasp.org
+    github: xeno6696
+level: 3
+license:
+  - BSD-3-Clause
+  - CC0-1.0
+  - MIT
+name: OWASP Enterprise Security API (ESAPI)
+pitch: Description for OWASP Enterprise Security API (ESAPI)
+repositories:
```

Comment:
Is the intent that we would be replicating this structure to each of the projects listed?